packages/fxa-settings/src/lib/auth-errors/en-US.ftl

```diff
@@ -3,6 +3,11 @@
 auth-error-102 = Unknown account
 auth-error-103 = Incorrect password
 auth-error-110 = Invalid token
+# This string is the amount of time required before a user can attempt another request.
+# Variables:
+#   $retryAfter (String) - Time required before retrying a request. This text is localized
+#                          by our server based on the accept language in request.
+#                           (for example: "15 minutes")
```

Comment:
I'm no l10n pro, but if the "minutes" part is in the variable, the localizer cannot do anything to it?

Reply:
oh sorry, I guess it could've been localized before it's used here. 👍